sql/9dbSummerJunctionRegistrationForm.sql

```diff
@@ -0,0 +1,74 @@
+CREATE TABLE dbSummerJunctionRegistrationForm (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    steam BOOLEAN,
+    summer_camp BOOLEAN,
+    child_first_name VARCHAR(50) NOT NULL,
+    child_last_name VARCHAR(50) NOT NULL,
+    birthdate DATE NOT NULL,
+    grade_completed INT NOT NULL,
+    gender ENUM('male', 'female') NOT NULL,
+    shirt_size ENUM('child-xs', 'child-s', 'child-m', 'child-l', 'child-xl', 
+                    'adult-s', 'adult-m', 'adult-l', 'adult-xl', 'adult-2x') NOT NULL,
+    neighborhood VARCHAR(100) NOT NULL,
+    child_address VARCHAR(100) NOT NULL,
+    child_city VARCHAR(50) NOT NULL,
+    child_state CHAR(2) NOT NULL,
+    child_zip CHAR(5) NOT NULL,
+    child_medical_allergies VARCHAR(255),
+    child_food_avoidances VARCHAR(255),
+
+    parent1_first_name VARCHAR(50) NOT NULL,
+    parent1_last_name VARCHAR(50) NOT NULL,
+    parent1_address VARCHAR(100) NOT NULL,
+    parent1_city VARCHAR(50) NOT NULL,
+    parent1_state CHAR(2) NOT NULL,
+    parent1_zip CHAR(5) NOT NULL,
+    parent1_email VARCHAR(100) NOT NULL,
+    parent1_cell_phone VARCHAR(15) NOT NULL,
+    parent1_home_phone VARCHAR(15),
+    parent1_work_phone VARCHAR(15),
+
+    parent2_first_name VARCHAR(50),
+    parent2_last_name VARCHAR(50),
+    parent2_address VARCHAR(100),
+    parent2_city VARCHAR(50),
+    parent2_state CHAR(2),
+    parent2_zip CHAR(5),
+    parent2_email VARCHAR(100),
+    parent2_cell_phone VARCHAR(15),
+    parent2_home_phone VARCHAR(15),
+    parent2_work_phone VARCHAR(15),
+
+    emergency_contact1_name VARCHAR(50) NOT NULL,
+    emergency_contact1_relationship VARCHAR(50) NOT NULL,
+    emergency_contact1_phone VARCHAR(15) NOT NULL,
+
+    emergency_contact2_name VARCHAR(50),
+    emergency_contact2_relationship VARCHAR(50),
+    emergency_contact2_phone VARCHAR(15),
+
+    authorized_pickup_persons VARCHAR(255) NOT NULL,
+    not_authorized_pickup_persons VARCHAR(255),
+
```

Comment:
I think for authorized _pickup_persons and not_authorized_pickup_persons, we were going to use the dbAuthorized table, since listing people would make the dbSummerJunctionRegistrationForm not normalized